packages/components/src/components.ts

```diff
@@ -270,6 +270,12 @@ export { default as HdsTextCode } from './components/hds/text/code.ts';
 export { default as HdsTextDisplay } from './components/hds/text/display.ts';
 export * from './components/hds/text/types.ts';
 
+// Time
+export { default as HdsTime } from './components/hds/time/index.ts';
+export { default as HdsTimeSingle } from './components/hds/time/single.ts';
+export { default as HdsTimeRange } from './components/hds/time/range.ts';
+export * from './services/hds-time-types.ts';
```

Comment:
@alex-ju @KristinLBradley I know this PR is closed, but should this one have been declared in a new `packages/components/src/services.ts` file (to be created), similarly to what it's been done for helpers and modifiers?

Reply:
these types are shared by both the Time components and the Time service. since we generally export component-related types via this file, I thought it would be more expected to have it here. `packages/components/src/services.ts` exists, but is empty, as for helpers and modifiers,

Reply:
(I am asking because I am doing the same thing for theming)

Reply:
I can open a new PR to change it if we wanted to.

Reply:
No, not necessary (for me).